pkg/ctl/cmdutils/configfile.go

```diff
@@ -883,6 +883,14 @@ func NewGetLabelsLoader(cmd *Cmd, ngName string) ClusterConfigLoader {
 	return l
 }
 
+// NewGetClusterLoader will load config or use flags for 'eksctl get cluster(s)'
+func NewGetClusterLoader(cmd *Cmd) ClusterConfigLoader {
+	l := newCommonClusterConfigLoader(cmd)
+	l.flagsIncompatibleWithoutConfigFile.Delete("name")
```

Comment:
This has no effect as `flagsIncompatibleWithoutConfigFile` does not contain a `name` element.  
```suggestion

```

Reply:
Huh, it doesn't? I think I had to add this otherwise it wasn't working... I'll test again.

Reply:
Yeah, it doesn't. No code change is required as `newCommonClusterConfigLoader` already disallows using both `--name` and `--config-file`.